speechmatics/cli.py

```diff
@@ -341,6 +342,10 @@ def get_transcription_config(
     if args_auto_chapters or auto_chapters_config is not None:
         config["auto_chapters_config"] = AutoChaptersConfig()
 
+    arg_audio_events = args.get("audio_events")
```

Comment:
Should also allow audio events through config file

Reply:
good catch, done